components/SectionTextBackground/SectionTextBackground.tsx

```diff
@@ -0,0 +1,67 @@
+import { Variant } from "../Variant/Variant";
+import styled from "styled-components";
+
+const StyledSectionTextBackground = styled.section<Variant>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${({ variant }) =>
+    variant === "dark" ? "var(--me-color-gray-500)" : "var(--me-color-white)"};
+  color: ${({ variant }) =>
+    variant === "dark" ? "var(--me-text-inverted)" : "var(--me-text-gray-700)"};
+`;
+
+const StlyedHeadlineGroup = styled.div`
+  justify-content: flex-end;
+  align-self: flex-end;
+  padding-bottom: 5rem;
+  margin: 0.5rem;
+`;
+
+const StyledParagraph = styled.div`
+  justify-content: flex-end;
+  align-self: flex-end;
+  padding-left: 16rem;
+  padding-bottom: 5rem;
+  margin-right: 5rem;
+`;
+
+interface ContentPropsTwo extends Variant {
+  headlineGroup: { headlineSmall: string; headline: string };
+  paragraph: string;
+  variant: "bright" | "dark";
```

Comment:
Not sure if this is intended: You're overriding the variant property from Variant and declaring it as required. But in the component you're setting a default which means you're expecting variant to be optional. To my understanding line 32 is redundant.